code/modules/reagents/reagents/nanites.dm

```diff
@@ -196,24 +167,40 @@
 /datum/reagent/nanites/trauma_control_system/will_occur(mob/living/carbon/M, alien, var/location)
 	if(..() && ishuman(M))
 		var/mob/living/carbon/human/H = M
-		for(var/obj/item/organ/organ in H.organs) //Grab the organ holding the implant.
-			if(organ.damage > 0 && !BP_IS_ROBOTIC(organ))
-				return TRUE
-		for(var/obj/item/organ/organ in H.internal_organs) //SYZYGY Edit
-			if(organ.damage > 0 && !BP_IS_ROBOTIC(organ)) //SYZYGY Edit
+		for(var/obj/item/organ/organ in H.internal_organs) //Occulus Edit
+			if(organ.damage > 0 && !BP_IS_ROBOTIC(organ)) //Occulus Edit
 				return TRUE // SYZYGY Edit
 
 /datum/reagent/nanites/trauma_control_system/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(..() && ishuman(M))
 		var/mob/living/carbon/human/H = M
-		for(var/obj/item/organ/organ in H.organs) //Grab the organ holding the implant.
-			if (istype(organ, /obj/item/organ/external) && organ.damage > 0 && !BP_IS_ROBOTIC(organ))
-				organ.heal_damage((2 + organ.damage * 0.03)* effect_multiplier, (2 + organ.damage * 0.03)* effect_multiplier)
-			//else if (istype(organ, /obj/item/organ/internal) && organ.damage > 0 && !BP_IS_ROBOTIC(organ)) - SYZYGY Edit: Fix internal organs
-			//	organ.heal_damage((2 + organ.damage * 0.03)* effect_multiplier) - SYZY EDIT : Fix internal organs
-		for(var/obj/item/organ/organ in H.internal_organs) //SYZYGY EDIT - Grab Internal Organs
-			if((organ.damage > 0) && !BP_IS_ROBOTIC(organ)) //SYZYGY Edit
-				organ.heal_damage(((0.2 + organ.damage * 0.03) * effect_multiplier), FALSE) //SYZYGY Edit
+		for(var/obj/item/organ/organ in H.internal_organs) //Occulus EDIT - Grab Internal Organs
+			if((organ.damage > 0) && !BP_IS_ROBOTIC(organ)) //Occulus Edit
+				organ.heal_damage(((0.2 + organ.damage * 0.03) * effect_multiplier), FALSE) //Occulus Edit
+
+/*Implantoids - Repairs synthetic organ damage*/
+
+/datum/reagent/nanites/implant_medics
+	name = "Implantoids"
+	id = "implant nanites"
+	description = "Microscopic construction robots programmed to repair prosthetics."
+
+/datum/reagent/nanites/implant_medics/will_occur(mob/living/carbon/M, alien, var/location)//Occulus Edit Start
+	if(..() && ishuman(M))
+		var/mob/living/carbon/human/H = M
+		for(var/obj/item/organ/organ in H.internal_organs) //Check Internal Organs
+			if(organ.damage > 0 && BP_IS_ROBOTIC(organ))
+				return TRUE//Occulus Edit
+
+/datum/reagent/nanites/implant_medics/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(..() && ishuman(M))
+		var/mob/living/carbon/human/H = M
+		for(var/obj/item/organ/organ in H.internal_organs) //The location of internal organs changed
+			if((organ.damage > 0) && BP_IS_ROBOTIC(organ))
+				organ.heal_damage(((0.2 + organ.damage * 0.03) * effect_multiplier), FALSE)
+
+/*Purgers - Purges nanites from the bloodstream, except themselves*/
+
```

Comment:
You can remove the specific occulus edits in here, and cover the entire block with a `Occulus Edit Start/End`